awx/api/views/__init__.py

```diff
@@ -403,6 +404,32 @@ class InstanceInstanceGroupsList(InstanceGroupMembershipMixin, SubListCreateAtta
     relationship = 'rampart_groups'
 
 
+class InstanceHealthCheck(GenericAPIView):
+
+    name = _('Instance Health Check')
+    model = models.Instance
+    serializer_class = serializers.InstanceHealthCheckSerializer
+    permission_classes = (IsSuperUser,)
```

Comment:
Is this saying that only super users will be able to retrieve health check data with a GET request.  System Auditors will also have access to the instance groups and instances list in the UI so it might be helpful to allow them to retrieve this data as well.

Reply:
That's valid.

I will plan to allow superusers to GET and POST, and allow system auditors to only GET.